extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -139,9 +139,11 @@ export class LocalSubscriptionPlatform {
         if (action == 'login') {
           // The "login" action is somewhat special b/c viewers can
           // enhance this action, e.g. to provide save/link feature.
-          const platform = this.serviceAdapter_.selectPlatformForLogin();
-          this.serviceAdapter_.delegateActionToService(
-              action, platform.getServiceId());
+          this.serviceAdapter_.selectPlatformForLogin()
```

Comment:
Ditto: it was synchronous due to popup blocker risks.

Reply:
see above